src/com/lundekhan/stokk/StokkRoute.kt

```diff
@@ -0,0 +1,83 @@
+package com.lundekhan
+
+import com.lundekhan.gui.HtmlTemplates.respondHtmlShell
+import com.lundekhan.stokk.StokkHelper
+import io.ktor.application.call
+import io.ktor.request.receiveParameters
+import io.ktor.routing.Route
+import io.ktor.routing.get
+import io.ktor.routing.post
+import io.ktor.routing.route
+import kotlinx.html.*
+import kotlin.math.roundToInt
+
+fun Route.stokkRoute(): Route = route("/stokk") {
+
+    fun SECTION.stokkForm(
+        year: String = "",
+        startAmount: String = "",
+        interest: String = "",
+        monthlyDeposit: String = ""
+    ): Unit = form(method = FormMethod.post) {
+        header {
+            style = "padding:0;"
+            h3 { +"compoundcalc." }
+            small { +"Calculate the compound interest!" }
+        }
+        section {
+            numberInput(name = "year") {
+                placeholder = "year"
+                if (year.isNotEmpty()) value = year
+            }
+            numberInput(name = "startAmount") {
+                min = "0"
+                step = "500"
+                placeholder = "startAmount"
+                if (startAmount.isNotEmpty()) value = startAmount
+            }
+        }
+        section {
+            numberInput(name = "interest") {
+                min = "0"
+                step = "0.1"
+                placeholder = "interest"
```

Comment:
`placeholder = "Average interest (7 = 7%)"`. (or `"Interest (7 = 7%)"`)

I think it makes sense to clarify how the interest is evaluated.